core/equations.gms

```diff
@@ -417,47 +417,49 @@ q_limitProd(t,regi,teRe2rlfDetail(teReNoBio(te),rlf))..
 q_limitGeopot(t,regi,peReComp(enty),rlf)..
   p_datapot(regi,"limitGeopot",rlf,enty)
   =g=
-  sum(te$teReComp2pe(enty,te,rlf), (vm_capDistr(t,regi,te,rlf) / (pm_data(regi,"luse",te)/1000)));
+  sum(te$teReComp2pe(enty,te,rlf), (v_capDistr(t,regi,te,rlf) / (pm_data(regi,"luse",te)/1000)));
 
-***  learning curve for investment costs
-***  deactivate learning for tech_stat 4 technologies before 2025 as they are not built before
+***---------------------------------------------------------------------------
+*' Learning curve for investment costs:
+*' (deactivate learning for tech_stat 4 technologies before 2025 as they are not built before)
+***---------------------------------------------------------------------------
 q_costTeCapital(t,regi,teLearn)$(NOT (pm_data(regi,"tech_stat",teLearn) eq 4 AND t.val le 2020)) ..
   vm_costTeCapital(t,regi,teLearn)
   =e=
-***  special treatment for first time steps: using global estimates better
-***  matches historic values
-    ( fm_dataglob("learnMult_wFC",teLearn)
+*** floor costs defined regionally
+  pm_data(regi,"floorcost",teLearn)
+*** until 2005: using global estimates better matches historic values
+  + ( fm_dataglob("learnMult_wFC",teLearn)
     * ( ( sum(regi2, vm_capCum(t,regi2,teLearn))
           + pm_capCumForeign(t,regi,teLearn)
         )
         ** fm_dataglob("learnExp_wFC",teLearn)
       )
     )$( t.val le 2005 )
-***  special treatment for 2010, 2015: start divergence of regional values by using a
-***  t-split of global 2005 to regional 2020 in order to phase-in the observed 2020 regional
-***  variation from input-data
-  + ( (2020 - t.val)/15 * fm_dataglob("learnMult_wFC",teLearn)
+*** 2005 to 2020: linear transition from global 2005 to regional 2020
+*** to phase-in the observed 2020 regional variation from input-data
+  + ( (2020 - t.val) / (2020-2015) * fm_dataglob("learnMult_wFC",teLearn)
```

Comment:
Why did you replace 15 by 2020-2015 = 5?

Reply:
Just to hide some bugs for more fun in the future 🤷 

It's a mistake thanks for digging and finding it!